pkg/spoke/spokeagent.go

```diff
@@ -57,6 +57,7 @@ func (o *WorkloadAgentOptions) AddFlags(cmd *cobra.Command) {
 	flags.Float32Var(&o.QPS, "spoke-kube-api-qps", o.QPS, "QPS to use while talking with apiserver on spoke cluster.")
 	flags.IntVar(&o.Burst, "spoke-kube-api-burst", o.Burst, "Burst to use while talking with apiserver on spoke cluster.")
 	flags.DurationVar(&o.StatusSyncInterval, "status-sync-interval", o.StatusSyncInterval, "Interval to sync resource status to hub.")
+	flags.BoolVar(&o.EnableExecutorCaches, "enable-executor-caches", o.EnableExecutorCaches, "Whether to enable the executor caches function.")
```

Comment:
Should we set this as a feature gate? so users can use klusterlet to config.
@qiujian16 WDYT?

Reply:
yes, I think we should do that.